scripts/variantstore/wdl/GvsSitesOnlyVCF.wdl

```diff
@@ -35,49 +59,150 @@ workflow GvsSitesOnlyVCF {
-           service_account_json_path = service_account_json_path,
-       }
+        call PrepAnnotationJson {
+          input:
+            annotation_json = AnnotateVCF.annotation_json,
+            output_file_suffix = "${i}.json.gz",
+            output_path = output_path,
+            service_account_json_path = service_account_json_path
+        }
     }
 
-     call BigQueryLoadJson {
-         input:
-             nirvana_schema = vat_schema_json_file,
-             vt_schema = variant_transcript_schema_json_file,
-             genes_schema = genes_schema_json_file,
-             project_id = project_id,
-             dataset_name = dataset_name,
-             output_path = output_path,
-             table_suffix = table_suffix,
-             service_account_json_path = service_account_json_path,
-             prep_jsons_done = PrepAnnotationJson.done
-         }
-
-     call BigQuerySmokeTest {
-         input:
-             project_id = project_id,
-             dataset_name = dataset_name,
-             counts_variants = ExtractAnAcAfFromVCF.count_variants,
-             table_suffix = table_suffix,
-             service_account_json_path = service_account_json_path,
-             load_jsons_done = BigQueryLoadJson.done
-         }
+    call BigQueryLoadJson {
+       input:
+         nirvana_schema = vat_schema_json_file,
+         vt_schema = variant_transcript_schema_json_file,
+         genes_schema = genes_schema_json_file,
+         project_id = project_id,
+         dataset_name = dataset_name,
+         output_path = output_path,
+         table_suffix = table_suffix,
+         service_account_json_path = service_account_json_path,
+         prep_jsons_done = PrepAnnotationJson.done
+  }
+
+    call BigQuerySmokeTest {
+       input:
+         project_id = project_id,
+         dataset_name = dataset_name,
+         counts_variants = ExtractAnAcAfFromVCF.count_variants,
+         table_suffix = table_suffix,
+         service_account_json_path = service_account_json_path,
+         load_jsons_done = BigQueryLoadJson.done
+   }
 }
 
 ################################################################################
+
+task GetSubpopulationCalculations {
+    input {
+        File input_vcf
+        File input_vcf_index
+        File sample_list
+        String subpopulation
+    }
+    String output_filename = subpopulation + ".vcf"
+    String output_vcf_idx = output_filename + ".idx"
+    String subpopulation_sample_list = "~{subpopulation}.args"
+
+    parameter_meta {
+        input_vcf: {
+            localization_optional: true
+        }
+        input_vcf_index: {
+            localization_optional: true
+        }
+    }
+    command <<<
+        set -e
+
+        # get all the sub-sample lists for each subpopulation
+        grep ~{subpopulation} ~{sample_list} | cut -d " " -f1 > ~{subpopulation_sample_list}
+
+        ## TODO we need a check that if there's NOTHING in this subpop we dont bother with the rest of this!
```

Comment:
Just wondering, is there any reason not to do this in this PR?

Reply:
nope---just wanted to ask the team the best way to do this and no one was around when I was writing it!